src/test/java/org/apache/maven/shared/utils/io/FileUtilsTest.java

```diff
@@ -441,9 +449,300 @@ public void copyFile1()
             testFile1.lastModified() == destination.lastModified());*/
     }
 
+    private static long MILLIS_PER_DAY = 24 * 60 * 60 * 1000L;
```

Comment:
Why not use TimeUnit for this?

Reply:
Happy to use `TimeUnit.DAYS.toMillis(1)` instead. Assumed `ChronoUnit` and friends weren't available but had forgotten `TimeUnit`!